packages/components/Checkbox/src/Checkbox.settings.old.ts

```diff
@@ -0,0 +1,95 @@
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import { checkboxName, ICheckboxType } from './Checkbox.types';
+
+export const checkboxSelectActionLabel = 'Toggle the Checkbox';
+
+export const settings: IComposeSettings<ICheckboxType> = [
+  {
+    tokens: {
+      borderColor: 'menuItemText',
+      color: 'menuItemText',
+      backgroundColor: 'menuBackground',
```

Comment:
Since this is win32 specific, would it be appropriate to make this settings.win32.ts?

Reply:
windows too !

Reply:
Ah. I don't really like the name "old" here.. but I get why you're doing it. I think I'd rather it just be ".windows" and for .win32 you just import the windows settings. Or "settingsWithoutSVG" to make it more descriptive.